src/figuresClasses.ts

```diff
@@ -1,19 +1,95 @@
-export interface Figure {
+enum GetShape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle'
+}
+
+enum GetColor {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue'
+}
 
+export interface Figure {
+  shape: GetShape;
+  color: GetColor;
+  getArea(): number;
 }
 
-export class Triangle {
+function getInvalidSides(...args: number[]): boolean {
```

Comment:
this function doesn't return invalid sides
will be better to name it somehow using `check` action

Reply:
Thanks. Changed)